faraday-raise-errors.gemspec

```diff
@@ -20,6 +20,6 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "faraday"
```

Comment:
We should probably pin this to a specific version now

Reply:
I can pin it to ">= 0.9.0", and I think we should be good with that, as it looks like the change to double-up the constants is since that version (back in 2012!): https://github.com/lostisland/faraday/commit/a6c6f1612c225335bf3bf94cf0b88e2e9a315671